routes/api/money.js

```diff
@@ -0,0 +1,139 @@
+const router= require('express').Router();
+
+const { Event, money, Op }= require('../../databases/db');
+
+////endpoint para listar eventos en los que pueda participar el usuario(en los que no esté participando)
+router.get('/', async (req,res)=>{
+
+    const eventsPublics= await Event.findAll({
+        attributes: {
+            exclude: ['createdAt', 'updatedAt', 'status']
+         },
+        raw:true,
+        where:{"status":true}
+    });
+     if(eventsPublics==''){
+        res.json({error:'No existen eventos activos'});
+    }else{
+        const eventUser = eventsPublics.map(async function(element) {
+            const moneyEvent = await money.findOne({
+                raw: true,
+                where: {
+                    [Op.and]: [{
+                        eventId: element.id
+                    },
+                    {
+                        userId: req.user.id.id
+                    }]
+                }
+            });
+            if (!moneyEvent) {
+                element.imAlreadyPlayingIn = false;
+            } else {
+                element.imAlreadyPlayingIn = true;
+            }
+            return element;
```

Comment:
Propuesta mas elegante
```suggestion
            });
            return {
                 ...element,
                 imAlreadyPlayingIn: !!moneyEvent,
            };
```

Reply:
Ok dejemos esto para despues